cmd/builder/internal/builder/config.go

```diff
@@ -38,6 +44,8 @@ type Config struct {
 	Connectors   []Module     `mapstructure:"connectors"`
 	Replaces     []string     `mapstructure:"replaces"`
 	Excludes     []string     `mapstructure:"excludes"`
+
+	downloadModules retry `mapstructure:"-"`
```

Comment:
I am not against this refactor, but we can do it on a separate PR, we don't need to do it here